waggle-dance-api/src/test/java/com/hotels/bdp/waggledance/api/model/FederatedMetaStoreConstructorTest.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (C) 2016-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.waggledance.api.model;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+public class FederatedMetaStoreConstructorTest extends AbstractMetaStoreTest<FederatedMetaStore> {
```

Comment:
Not sure if you wanted me to review this yet but why aren't these tests just in the FederatedMetaStore test?